packages/twenty-front/src/modules/apollo/optimistic-effect/types/OptimisticEffectSharedTypes.ts

```diff
@@ -0,0 +1,26 @@
+import { type ApolloCache } from '@apollo/client';
+
+import { type ObjectMetadataItem } from '@/object-metadata/types/ObjectMetadataItem';
+import { type RecordGqlNode } from '@/object-record/graphql/types/RecordGqlNode';
+import { type ObjectRecord } from '@/object-record/types/ObjectRecord';
+import { type ObjectPermissions } from 'twenty-shared/types';
+
+// Common context passed to all optimistic effects
+export type OptimisticEffectContext = {
```

Comment:
one file per type, + name Context is not good +should contain OptimistifEffect in CreateEdgeParams